environment/src/main/java/com/sequenceiq/environment/parameters/dao/domain/AwsParameters.java

```diff
@@ -12,7 +18,7 @@
 
 @Entity
 @DiscriminatorValue("AWS")
-public class AwsParameters extends BaseParameters {
+public class AwsParameters extends BaseParameters implements AccountIdAwareResource {
```

Comment:
is this change really needed?

Reply:
Yes, This is needed, Without this the Environment creation is failing with the error "AwsParameters must be a subclass of AccountIdAwareResource"